### ISSUE TITLE
Installing NSR in CrDroid 8.x mangles System UI text

### ISSUE THREAD

**Issue by midaspt** [DanGLES3/NoStorageRestrict#4]
Please check my posts at _XDA-Devs_ for a report and screenshots: https://forum.xda-developers.com/posts/87083901.

Will check back and try to assist if needed...

**Comment by DanGLVK** [comment]
> Please check my posts at _XDA-Devs_ for a report and screenshots: https://forum.xda-developers.com/posts/87083901.
> 
> Will check back and try to assist if needed...

That's odd, could you provide some logs?

**Comment by midaspt** [comment]
Very sorry, it seems I was too fast in jumping to this conclusion -- I was so relieved to have found a solution. 😞 

But after I installed the _Audio Modification Library_ module, the problem returned, so the issue isn't specific of your module...

In any case what logs do you require? **Magisk**, _dmesg_ or what?  Please tell me how to get the logs, as my knowledge of Android/Linux is a little fuzzy...

**Comment by DanGLVK** [comment]
> Very sorry, it seems I was too fast in jumping to this conclusion -- I was so relieved to have found a solution. 😞
> 
> But after I installed the _Audio Modification Library_ module, the problem returned, so the issue isn't specific of your module...
> 
> In any case what logs do you require? **Magisk**, _dmesg_ or what? Please tell me how to get the logs, as my knowledge of Android/Linux is a little fuzzy...

Logcat will do, plenty of tutorials on how to get them

**Comment by midaspt** [comment]
Right, extracted log after reinstalling module and rebooting with **Matlog**, let's see if this is what's needed: mir.cr/MTG5AVPP...

A lot of errors, apparently, because the device had no inserted SIMs, but wireless was working.
